src/test/java/fi/metatavu/metaform/server/test/functional/tests/ReplyTestsIT.kt

```diff
@@ -613,15 +617,20 @@ class ReplyTestsIT : AbstractTest() {
     @Test
     @Throws(Exception::class)
     fun testExportReplyPdf() = TestBuilder().use { testBuilder ->
+        val metaformNoTheme: Metaform = testBuilder.systemAdmin.metaforms.createFromJsonFile("simple")
+
         val exportTheme = testBuilder.systemAdmin.exportThemes.createSimpleExportTheme()
         testBuilder.systemAdmin.exportFiles.createSimpleExportThemeFile(exportTheme.id!!, "reply/pdf.ftl", "<!DOCTYPE html><html><head><meta charset=\"UTF-8\"></meta><title>title</title></head><body>content</body></html>")
-        val metaform: Metaform = testBuilder.systemAdmin.metaforms.createFromJsonFile("simple")
-        val newMetaform = Metaform(metaform.id, metaform.visibility, exportTheme.id, metaform.allowAnonymous,
-                metaform.allowDrafts, metaform.allowReplyOwnerKeys, metaform.allowInvitations, metaform.autosave,
-                metaform.title, metaform.slug, metaform.sections, metaform.filters, metaform.scripts)
-        testBuilder.systemAdmin.metaforms.updateMetaform(newMetaform.id!!, newMetaform)
-        val reply: Reply = testBuilder.test1.replies.createSimpleReply(metaform.id!!, "Test 1, Ääkköstesti ÅÅ, Правда", ReplyMode.UPDATE)
-        assertPdfDownloadContents("content", testBuilder.systemAdmin.token, metaform, reply)
+        val simple2 = testBuilder.systemAdmin.metaforms.createFromJsonFile("simple")
+        val metaformWithTheme = testBuilder.systemAdmin.metaforms.updateMetaform(simple2.id!!, simple2.copy(exportThemeId = exportTheme.id))
+
+        // export as pdf with base theme
+        val replyWithoutTheme: Reply = testBuilder.test1.replies.createSimpleReply(metaformNoTheme.id!!, "Test 1, Ääkköstesti ÅÅ, Правда", ReplyMode.UPDATE)
+        assertPdfDownloadNotContents("content", testBuilder.systemAdmin.token, metaformNoTheme, replyWithoutTheme)
```

Comment:
Why does it check for text "content" ?

Reply:
Just to show that "content" from the other theme did not get applied here unlike it did several lines lower. I added a comment about it.